cmd/countdown.go

```diff
@@ -108,7 +108,11 @@ func runCountdown(args []string) (cdTimes int, err error) {
 			case <-ticker.C:
 				ec, _ = ec.Next()
 				cdTimes++
-				log.PrintColor(logger.Yellow, fmt.Sprintf("%d days %d hours %d minutes %d seconds\r", ec.Days, ec.Hours, ec.Minutes, ec.Seconds))
+				seconds := fmt.Sprintf("%d", ec.Seconds)
+				if ec.Seconds < 10 {
+					seconds = "0" + seconds
+				}
+				log.PrintColor(logger.Yellow, fmt.Sprintf("%d days %d hours %d minutes %s seconds\r ", ec.Days, ec.Hours, ec.Minutes, seconds))
```

Comment:
We can format 2 to 02 :) 
```suggestion
				log.PrintColor(logger.Yellow, fmt.Sprintf("%d days %d hours %02d minutes %02d seconds\r ", ec.Days, ec.Hours, ec.Minutes, ec.Seconds))
```